SEED-platform/seed
# Sync sensor readings creation

#### Any background context you want to provide?
 This syncs sensor readings creation to speed it up and gives better error messages.
#### What's this PR do?

#### How should this be manually tested?
try uploading `seed/tests/data/small_sensor_readings_example_bad.csv` to see erorrs.
try uploading `seed/tests/data/sensor_readings_example.csv`to see progress bar and speed.
#### What are the relevant tickets?

#### Screenshots (if appropriate)
